src/black/__init__.py

```diff
@@ -515,8 +520,21 @@ def get_sources(
     stdin_filename: Optional[str],
 ) -> Set[Path]:
     """Compute the set of files to be formatted."""
+    root, method = find_project_root(src)
+
+    if verbose:
+        if method:
+            out(
+                f"Identified `{root}` as project root containing a {method}.",
+                fg="blue",
+            )
+        else:
+            out(f"Identified `{root}` as project root.", fg="blue")
+        paths = '", "'.join(
+            str(Path(source).resolve().relative_to(root)) for source in src
+        )
+        out(f'Sources to be formatted: "{paths}"', fg="blue")
```

Comment:
```suggestion
        normalized_paths = [Path(source).resolve().relative_to(root) for source in src]
        srcs_string = '", "'.join(normalized_paths)
        out(f'Sources to be formatted: "{srcs_string}"', fg="blue")
```
This also makes it harder for future developers to mistake `paths` for `src`.